BigQuery/tests/Snippet/BigQueryClientTest.php

```diff
@@ -412,6 +412,23 @@ public function testTimestamp()
 
         $this->assertInstanceOf(Timestamp::class, $res->returnVal());
     }
+
+    public function testGetServiceAccount()
+    {
+        $expectedEmail = uniqid() . '@bigquery-encryption.iam.gserviceaccount.com';
+        $snippet = $this->snippetFromMethod(BigQueryClient::class, 'getServiceAccount');
+        $snippet->addLocal('bigQuery', $this->client);
+        $this->connection->getServiceAccount(['projectId' => $this::PROJECT_ID])
```

Comment:
Not really a big deal, so feel free to leave as is - was just curious why we have `$this` notation here and `self` in the unit test?

Reply:
Done